docs/model_server_c_api.md

```diff
@@ -11,6 +11,12 @@ Server functionalities are encapsulated in shared library built from OVMS source
 
 Calling a method to start the model serving in your application initiates the OVMS as a separate thread. Then you can schedule inference both directly from app using C API and gRPC/HTTP endpoints.
 
+API is versioned according to [SemVer 2.0](https://semver.org/). Calling `OVMS_ApiVersion` it is possible to get `major` and `minor` version number.
+- major - incremented when new, backward incompatible changes are introduced to the API itself (API call removal, name change, parameter change)
+- minor - incremented when API is modified but backward compatible (new API call added)
+
+There is no patch version number. Underlying functionality changes not related to API itself are tracked via OVMS version. OVMS and OpenVINO versions can be tracked via logs or `ServerMetadata` request (via KServe or C-API).
```

Comment:
There is no ServerMetadata call in C-API now

Reply:
Beside that looks good

Reply:
fixed